package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "turbo dev",
     "build": "turbo build",
     "type-check": "turbo type-check",
-    "changeset:publish": "changeset publish"
+    "changeset:publish": "pnpm build && changeset publish"
```

Comment:
@IzumiSy 
I wonder if it would be better to follow the npm lifecycle. What do you think?
https://docs.npmjs.com/cli/v11/using-npm/scripts#npm-publish

Reply:
I didn't know npm has those useful hooks! We better follow that.

Let me add one more change to use that...